### PR TITLE
[Localization] Fix french translations

### PR DESCRIPTION
**Pull request by Nolway**
## What are the changes?
Translate missing translation for the french language.

## Why am I doing these changes?
Because a lot of text isn't translated.

## What did change?
A lot of sentences and words are translated in french.

### Screenshots/Videos
image
image

## How to test the changes?
Select the french language and go in-game.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/ability-trigger.ts [hunk 1]
```diff
@@ -4,5 +4,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "blockRecoilDamage" : "{{abilityName}}\nde {{pokemonName}} le protège du contrecoup !",
   "badDreams": "{{pokemonName}} a le sommeil agité !",
   "windPowerCharged": "{{pokemonName}} a été touché par la capacité {{moveName}} et se charge en électricité !",
-  "iceFaceAvoidedDamage": "{{pokemonName}} avoided\ndamage with {{abilityName}}!"
+  "perishBody": "{{abilityName}} de {{pokemonName}} \névanouira les deux pokémons en 3 tours !",
```
Comment (Greenlamp2) [inline src/locales/fr/ability-trigger.ts:7]: Mauvaise traduction ici.

File: src/locales/fr/achv.ts [hunk 2]
```diff
@@ -171,98 +171,98 @@ export const PGMachv: AchievementTranslationEntries = {
...
   "MONO_BUG": {
-    name: "Sting Like A Beedrill",
+    name: "Une chenille !!!",
   },
   "MONO_GHOST": {
     name: "Who you gonna call?",
   },
   "MONO_STEEL": {
-    name: "Mono STEEL",
+    name: "Acier galvanisé",
```
Comment (Greenlamp2) [inline src/locales/fr/achv.ts:238]: J'aime beaucoup cette ref

**Review by Greenlamp2 — CHANGES_REQUESTED** [review]

File: src/locales/fr/ability-trigger.ts (hunk 1 above)
Comment (Nolway) [reply src/locales/fr/ability-trigger.ts:7]: Ah oui my bad je fix ça

Comment (Nolway) [reply src/locales/fr/ability-trigger.ts]: Fixed

File: src/locales/fr/ability-trigger.ts [hunk 3]
```diff
@@ -4,5 +4,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "blockRecoilDamage" : "{{abilityName}}\nde {{pokemonName}} le protège du contrecoup !",
   "badDreams": "{{pokemonName}} a le sommeil agité !",
   "windPowerCharged": "{{pokemonName}} a été touché par la capacité {{moveName}} et se charge en électricité !",
-  "iceFaceAvoidedDamage": "{{pokemonName}} avoided\ndamage with {{abilityName}}!"
+  "perishBody": "{{abilityName}} de {{pokemonName}} \nmettra KO les deux pokémons dans 3 tours !",
+  "poisonHeal": "{{abilityName}} de {{pokemonName}} \nrétablit un peu ses HP !",
```
Comment (Greenlamp2) [inline src/locales/fr/ability-trigger.ts:8]: Du coup HP doit être traduit aussi

**Review by Greenlamp2 — APPROVED** [review]

**Pull request merged**